babel/src/main/java/org/apache/calcite/sql/babel/SqlUsingRequestModifier.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.jdbc.CalcitePrepare;
+import org.apache.calcite.sql.SqlCall;
```

Comment:
While this import list looks totally fine, I would suggest you to also consider using wildcard character to avoid the long import list. Using wildcard import could reduce true dependencies and therefore would keep the module less coupled.

Reply:
It looks like wildcard imports do not adhere to the Google java style guide: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Good point! Thank you for pointing this out.